src/yunohost/app.py

```diff
@@ -261,17 +261,16 @@ def app_list(filter=None, raw=False, installed=False, with_backup=False):
     # Get app list from the app settings directory
     for app in os.listdir(APPS_SETTING_PATH):
         if app not in app_dict:
-            # Handle multi-instance case like wordpress__2
-            if '__' in app:
-                original_app = app[:app.index('__')]
-                if original_app in app_dict:
-                    app_dict[app] = app_dict[original_app]
-                    continue
-                # FIXME : What if it's not !?!?
+            app_id, app_instance_nb = _parse_app_instance_name(app)
+            # Handle multi-instance case
+            if app_instance_nb > 1:
+                app_dict[app] = app_dict[app_id]
 
-            with open(os.path.join(APPS_SETTING_PATH, app, 'manifest.json')) as json_manifest:
-                app_dict[app] = {"manifest": json.load(json_manifest)}
+        with open(os.path.join(APPS_SETTING_PATH, app, 'manifest.json')) as json_manifest:
+            app_dict[app]['manifest'] = json.load(json_manifest)
```

Comment:
So if I understand correctly, the main fix comes from the unindent here ? Like, `app_dict[app]` is already supposed to be filled with the manifest from the applist but we override it with the one from the setting directory ?

I'm a bit afraid that this is not the right fix, as it means that for instance, for multi-instance app (e.g. installing a second wordpress) you would fill the manifest corresponding to the already-installed version of the wordpress ... Which may actually conflict with the version of the wordpress app that will actually be instsalled, and may create various issues ...

I think instead we should just explicitly add a new key somehow .... probably in the `status` section which also contains info like `installed_at` and `upgraded_at`.

Reply:
Oups, in fact it's not this part of app.py that used for the app_info.ms... Do I have to close and open a new PR to fix it? Or can I force-push?